Externals/Bochs_disasm/Bochs_disasm.vcxproj

```diff
@@ -1,23 +1,31 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|ARM64">
+      <Configuration>Debug</Configuration>
+      <Platform>ARM64</Platform>
+    </ProjectConfiguration>
     <ProjectConfiguration Include="Debug|x64">
       <Configuration>Debug</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|ARM64">
+      <Configuration>Release</Configuration>
+      <Platform>ARM64</Platform>
+    </ProjectConfiguration>
     <ProjectConfiguration Include="Release|x64">
       <Configuration>Release</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{8ADA04D7-6DB1-4DA4-AB55-64FB12A0997B}</ProjectGuid>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
Note: this requires a recent MSBuild (and/or Visual Studio 2019?) Version to work correctly. This basically means "use the latest available 10.0 SDK" while not limiting it to a specific build; but older versions of MSBuild/VS don't support this and simply fail with "Cannot find Windows SDK version 10.0" instead.
I _think_ this needs at least 16.3, but I don't remember exactly if it was _just_ any 16.x (since I've seen similar failures while attempting to do this with VS 2019 on the developer machine but VS 2017 build tools on the CI builders)

This ofc. goes for all the projects, not just this one.

Reply:
The builders have been updated to VS2019, so I don't think this should be a problem.

Reply:
Good to know. But now that you mention this; I _think_ we also need to update the developer requirements in our README. Or does this still build on VS 2017 without installing additional things?